web/src/containers/Sidebar.js

```diff
@@ -1,219 +1,227 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
 import { t } from '../i18n';
 import { expandActivitySection } from '../actions/activities';
 import { saveApd } from '../actions/apd';
-import { toggleExpand as toggleExpandAction } from '../actions/sidebar';
 import Btn from '../components/Btn';
-import SidebarLink from '../components/SidebarLink';
+import VerticalNav from '@cmsgov/design-system-core/dist/components/VerticalNav/VerticalNav';
 
-const linkGroup1 = [
-  {
-    id: 'apd-state-profile',
-    name: t('apd.stateProfile.title'),
-    sub: [
+class Sidebar extends Component {
+  state = { selectedId: 'apd-state-profile-office' };
+
+  handleSelectClick = (id) => this.setState({ selectedId: id.id });
```

Comment:
@mgwalker I'm not entirely sure I understand what you're asking -- are you suggesting passing `id.id` into the function so that the state setter reads `setState({ selectedId: id })`? Or am I missing the point?